### PR TITLE
Add Water Pumping Station

### PR DESCRIPTION
**Pull request by adam-a-a**
## Summary/Motivation:
Add water pumping station to support PR#457 and provide ability that @TimBartholomew asked for.

## Changes proposed in this PR:
- add unit, yaml, and tests
- add config option "fix_pump_power" which is set to True by default.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by TimBartholomew — DISMISSED** [review]
LGTM, I'm adding an issue to our master list for after March that we may want to consider reorganizing all of the zero-order pump models

File: watertap/data/techno_economic/tests/test_unit_parameter_files.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def test_unit_parameter_files(tech):
                     "smp",
                     "sw_onshore_intake",
                     "tramp_oil_tank",
+                    "water_pumping_station"
```
Comment (bknueven) [inline watertap/data/techno_economic/tests/test_unit_parameter_files.py]:
```suggestion
                    "water_pumping_station",
```

File: watertap/data/techno_economic/tests/test_unit_parameter_files.py [hunk 2]
```diff
@@ -114,7 +115,8 @@ def test_unit_parameter_files(tech):
                                          "agglom_stacking",
                                          "storage_tank",
                                          "evaporation_pond",
-                                         "deep_well_injection"]
+                                         "deep_well_injection",
+                                         "water_pumping_station"]
```
Comment (bknueven) [inline watertap/data/techno_economic/tests/test_unit_parameter_files.py]:
```suggestion
                                         "water_pumping_station",
                                         ]
```

**Review by bknueven — DISMISSED** [review]
This looks reasonable.

I would strongly encourage the use of trailing commas on list-of-strings to avoid potential bugs. Note that:
```
["foo"
 "bar"]
```
is *not* a syntax error -- it results in `["foobar"]`, as opposed to `["foo", "bar"]`, which is likely what was intended. However, this mistake can be easy to make by forgetting to add a comma to the previously last element.

**Review by bknueven — APPROVED** [review]

**Review by luohezhiming — APPROVED** [review]

**Pull request merged**